site/diagrams/sequenceDiagrams/command_control_concept_one_way_http.txt

```diff
@@ -0,0 +1,15 @@
+participant Device
+participant HTTP Adapter
+participant Application
+
+Device->HTTP Adapter: 1. HTTP request ("POST", "/event", "hono-ttd=30")
+activate HTTP Adapter
+HTTP Adapter->Application: 2. transfer("event/TENANT","4711","ttd=30")
+activate Application
+Application->HTTP Adapter: 3. transfer("control/TENANT/4711","light","ON")
+HTTP Adapter->Application: disposition("accepted")
+HTTP Adapter->Device: 4. HTTP response (("200", "hono-cmd=light"), "ON")
```

Comment:
iagain, t is actually the other way around, first we send the response then we accept the transfer